crates/holochain/src/sweettest/sweet_conductor_config.rs

```diff
@@ -3,44 +3,256 @@ use std::sync::Arc;
 use holochain_conductor_api::{conductor::ConductorConfig, AdminInterfaceConfig, InterfaceDriver};
 use kitsune_p2p::KitsuneP2pConfig;
 
+tokio::task_local! {
+    static RENDEZVOUS: DynSweetRendezvous;
+}
```

Comment:
Looks like this is not being used anywhere, is it? Or does the mere existence of this uninitialized static have some effect?

Could it be just as good to just create one of these explicitly at the beginning of whatever task needs it?

Reply:
Aha, whoops! This was from an earlier attempt, will delete!